lib/build-view.js

```diff
@@ -158,10 +162,16 @@ export default class BuildView extends View {
     if (0 === w || 0 === h) {
       return;
     }
-
+    const orientation = atom.config.get('build.panelOrientation') || 'Bottom';
+    if ( orientation === 'Left' || orientation === 'Right' ) {
+      this.cache[orientation].width = $('.build .output').width();
+    } else if ( orientation === 'Top' || orientation === 'Bottom' ) {
+      this.cache[orientation].height = $('.build .output').height();
```

Comment:
Have committed a 'flex' approach, which seems to work, thanks! Still need to resize the terminal initially.

I've also slipped another fix in with this commit to make the build output window line up with the item-views when the panel is on the left or right. Basically the header is resized initially to be the same height as the tabs/padding above the editor window. Would you prefer that to be a separate pull request?